### ISSUE TITLE
Mapping the parameter space in EEG/MEG analyses

### ISSUE THREAD

**Issue by johalgermissen** [ohbm/hackathon2020#139]
**Guidelines**

*We are very excited to meet you at the 2020 OHBM Brainhack 🎉* *To submit a project, you need to be an attendee of the 2020 OHBM Brainhack. We ask you to register first over [here](http://www.humanbrainmapping.org/HackathonReg/). Thank you!*

*We have prepared a checklist to help with your project submission. Here is how to proceed:*
 1. *Before filling in any part, please submit this issue*
 2. *Check items in the checklist below as you go through them*
 3. *Once you are done (at least all 'required' items must be provided), please delete the "Guidelines" section add a comment saying 'hi @ohbm/project-monitors: My project is ready!'*

Thank you!

*After step 1 (issue submitted), we will assign a 'project monitor' to follow your submission. If at any time you need help or anything is unclear, please add a comment and ping your project monitor. Our team is here to help!*

----------------------------
## Project info

**Title**:

**Project lead**:

**[Timezone](https://github.com/ohbm/hackathon2020/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#timezone)**:

**Hub**:

**Description**:

**Link to project**:

**Mattermost handle**:

**Goals for the OHBM Brainhack**

**Good first issues**:

**Skills**:

**Chat channel**:

**Video channel**:

## Submission checklist
*Once the issue is submitted, please check items in this list as you add under 'Additional project info'*

Please include the following above (all required):
-   [ ] Link to your project: could be a code repository, a shared document, etc. See [here](https://github.com/ohbm/hackathon2020/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#link-to-project)
-   [ ] Include your Mattermost handle]. If you do not have an account, please [sign up here](https://mattermost.brainhack.org/signup_email).
-   [ ] Goals for the OHBM Brainhack: describe what you want to achieve during this brainhack. See [here](https://github.com/ohbm/hackathon2020/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#goals).
-   [ ] Flesh out at least 2 "good first issues": those are tasks that do not require any prior knowledge about your project, could be defined as issues in a GitHub repository, or in a shared document, cf [here](https://github.com/ohbm/hackathon2020/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#onboarding-2-good-first-issues).
-   [ ] Skills: list skills that would be particularly suitable for your project. We ask you to include at least one non-coding skill, cf. [here](https://github.com/ohbm/hackathon2020/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#onboarding-skills).
-   [ ] Chat channel: A link to a chat channel that will be used during the OHBM Brainhack. This can be an existing channel or a new one. We recommend using the [Brainhack space on mattermost](https://mattermost.brainhack.org/), cf. [here](https://github.com/ohbm/hackathon2020/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#chat).
-   [ ] Video channel: Please create a video channel that will be used during the OHBM Brainhack and share it in your chat channel above. This can be an existing channel or a new one. For instance a [jitsi meet](https://meet.jit.si/) room, cf. [here](https://github.com/ohbm/hackathon2020/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#video-calls).

You can also include information about (all optional):
-   [ ] Number of participants, cf. [here](https://github.com/ohbm/hackathon2020/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#participant-capacity)
-   [ ] Twitter-size summary of your project pitch, cf. [here](https://github.com/ohbm/hackathon2020/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#twitter-size-summary-of-your-project-pitch)
-   [ ] Provide an image of your project for the OHBM brainhack website
-   [ ] Set up a kanban board on your repository to better divide the work and keep track of things, cf [here](https://github.com/ohbm/hackathon2020/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#set-up-a-kanban-board)
-   [ ] Project snippet for the OHBM Brainhack website, cf. [here](https://github.com/ohbm/hackathon2020/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#project-snippet-for-the-ohbm-brainhack-website)

We would like to think about how you will credit and onboard new members to your project. We recommend reading references from [this section](https://github.com/ohbm/hackathon2020/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#credit-and-onboarding). If you'd like to share your thoughts with future project participants, you can include information about (recommended):
-   [ ] Specify how will you acknowledge contributions (e.g. listing members on a contributing page).
-   [ ] Provide links to onboarding documents if you have some.

**Comment by johalgermissen** [comment]
hi @ohbm/project-monitors: My project is ready!
(will add a picture and text for the brainhack webpage later today)

**Comment by johalgermissen** [comment]
Filed pull request to add picture and adapted text o brainhack webpage

**Comment by Remi-Gau** [comment]
> Filed pull request to add picture and adapted text o brainhack webpage

hey @johalgermissen 
Thanks for that.
I have just sent you a PR on top of that to fix a small indent issue in the yml file. If you merge it it should update this PR.

**Comment by Remi-Gau** [comment]
I will be following this project for the @ohbm/hack2020-social_media

**Comment by complexbrains** [comment]
@johalgermissen You indicated that you will share the video link tomorrow, so may I ask you to share your video link, when it is ready, only on your Mattermost channel please to avoid any unwanted interferences, and please pin the message to your channel, please? We look forward to meeting you tomorrow! 🤗

**Comment by johalgermissen** [comment]
@complexbrains yes, already saw the alert on Mattermost! Now also updated the project description here.

**Comment by complexbrains** [comment]
Thank you! Then I will complete the review of your project!

**Comment by complexbrains** [comment]
Dear @johalgermissen  Thank you very much for your project submission. Your project looks ready. Welcome aboard! 🤗 🎊

**Comment by carolyog** [comment]
Hi @johalgermissen! I found your project very interesting and I'd like to take part. However, I belong to the Americas hub (UTC-3, São Paulo, Brazil more precisely). so I'll be attending the Americas hub Traintrack. Would it be a problem?
Thank you!
